Mathlib/CategoryTheory/Functor/ReflectsIso.lean

```diff
@@ -71,6 +71,13 @@ instance reflectsIsomorphisms_of_comp (F : C ⥤ D) (G : D ⥤ E)
     haveI := isIso_of_reflects_iso (F.map f) G
     exact isIso_of_reflects_iso f F⟩
 
+lemma reflectsIsomorphisms_of_comp' (F : C ⥤ D) (G : D ⥤ E)
```

Comment:
arguably the previous lemma should be `reflectsIsomorphisms_comp` and this should be unprimed.

Reply:
Yes, indeed.